src/editor/components/CopyButton/CopyButton.tsx

```diff
@@ -0,0 +1,42 @@
+import React, {useState} from 'react';
```

Comment:
Could we use `ClipboardButton` from uikit instead?

Reply:
Yes, we could. Should look for it first